logging/pom.xml

```diff
@@ -34,6 +34,30 @@
             <artifactId>gson</artifactId>
             <version>2.8.6</version>
         </dependency>
+        <dependency>
```

Comment:
what's this being pulled in for?

Reply:
The LoggingFilter has a dependency on `org.jboss.logging.MDC`. but it really should be `org.slf4j.MDC`!